src/RichText/RichText.tsx

```diff
@@ -1,8 +1,7 @@
 import * as React from "react";
 import ReactDOM from "react-dom";
-import PropTypes from "prop-types";
 import extend from "lodash/extend";
-import throttle from "lodash/throttle";
+import { throttle, forOwn } from "lodash";
```

Comment:
Does such import really work? There's a different package https://www.npmjs.com/package/lodash-es that allows ES-style imports

Reply:
Yes, it does, however it's transpiled to es5 as 
```
var _lodash = require("lodash");
      .....
      _lodash.forOwn.....
```

Reply:
Still, if we later migrate to ES6 it could be a problem. What if you add lodash-es package instead?

Reply:
Ok. Leave it for now